lib/spark_api/models/saved_search.rb

```diff
@@ -24,9 +24,6 @@ def self.provided()
         Class.new(self).tap do |provided|
           provided.element_name = '/savedsearches'
           provided.prefix = '/provided'
-          def provided_search?
-            true
-          end
```

Comment:
This is inside the `SavedSearch.provided` method, but the snippet cuts that off. I think that any time `SavedSearch.provided` was called, it overwrote the `provided_search?` method for the whole SavedSearches class, and not just for provided searches. I shouldn't have been messing around here without full understanding what was happening.